hips/tiles/tests/test_surveys.py

```diff
@@ -0,0 +1,22 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.utils.data import get_pkg_data_filename
+from astropy.tests.helper import remote_data
+from ..surveys import HipsSurveyPropertyList
+
+class TestHipsSurveyPropertyList:
+    @classmethod
+    def setup_class(cls):
+        filename = get_pkg_data_filename('data/surveys.txt')
+        cls.hips_survey_list = HipsSurveyPropertyList.read(filename)
+
+    @remote_data
+    def test_fetch(self):
+        self.hips_survey_list.fetch()
+
+    def test_surveys(self):
+        len_precomp = 4
+
+        assert len(self.hips_survey_list.surveys) == len_precomp
+        assert self.hips_survey_list.surveys[0]['surveys']['creator_did'] == 'ivo://CDS/C/MUSE-M42'
```

Comment:
You could introduce a temp variable here to keep the assert statements nice and short:
```
survey = self.hips_survey_list.surveys[0]['surveys']
```
Also: accessing [0]['surveys'] is very weird, suggest to not put the data in a Table, but just a list of `HipsSurveyProperties` objects.

And please also add a similar assert for some entry other than the first one. E.g. for the last survey you have in that example file.